src/app/shared/navbar/navbar.html

```diff
@@ -1,6 +1,11 @@
 <!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
 <nav class="docs-navbar-header" aria-label="Top Toolbar"
      [class.is-next-version]="isNextVersion">
+  <div class="skip-link-wrapper" [class.cdk-visually-hidden]="skipLinkHidden" >
```

Comment:
I tried investigating this as well and couldn't find a way to make `routerLink` work. In general, that would be more desirable, but because the `NavBar` exists at the root, I couldn't figure out how to get the navigation to be relative to the final activated component rather than the root. Otherwise, you could use `<a [routerLink]="[]" fragment="main-content"></a>` or something similar to update the fragment.